dataservice/api/biospecimen/schemas.py

```diff
@@ -17,9 +17,15 @@
 )
 
 ANALYTE_TYPE_ENUM = {"DNA", "RNA", "Other", "Virtual"}
-SAMPLE_PROCUREMENT_ENUM = {"Autopsy", "Biopsy", "Subtotal Resections",
-                           "Gross Total Resections", "Blood Draw",
-                           "Bone Marrow Aspiration", "Other"}
+SAMPLE_PROCUREMENT_ENUM = {
+    "Autopsy", "Biopsy", "Subtotal Resections",
+    "Gross Total Resections", "Blood Draw",
+    "Bone Marrow Aspiration", "Other",
+    # Added for CBTN
+    "Surgery", "Blood Collection - proband", "Saliva Collection - proband",
+    "Blood Collection - maternal", "Saliva Collection - maternal",
```

Comment:
Per Slack:
capitalize the P in Proband
Change Surgery to Surgical Resections

Reply:
And also capitalize "m"s and "p"s for maternal and paternal?